test/mix/tasks/phx.gen.context_test.exs

```diff
@@ -102,7 +102,7 @@ defmodule Mix.Tasks.Phx.Gen.ContextTest do
         assert file =~ "def change_post"
       end
 
-      assert_file "test/blog_test.exs", fn file ->
+      assert_file "test/phoenix/blog/blog_test.exs", fn file ->
```

Comment:
@josevalim raised a question if this should be `test/phoenix/blog_test.exs`.

Even though `my_app/blog/blog_test.exs` does look kinda weird, personally, I like it better than the alternative. First of all, tooling will trivially work (just replace `lib/*.ex` with `test/*_test.exs`). Secondly, if there will be more files within the context directory e.g. `test/my_app/blog/post_test.exs`, `test/my_app/blog/comment_test.exs` etc, I think it would make more sense to keep the together wishing `test/my_app/blog/`